backend/tournesol/resources/export_readme.txt

```diff
@@ -18,43 +18,116 @@ List of files
 represented by a rating given by a user for a specific criteria and a pair of
 videos.
 
+List of columns:
+
+- public_username:
+  The username of the user who submitted the comparison.
+- video_a:
+  One of the two video being compared. If the score is negative, it indicates
+  that the user prefers video_a against video_b.
+- video_b:
+  The second video being compared. If the score is positive, it indicates that
+  the user prefers video_b against video_a.
+- criteria:
+  The name of the criteria for the comparison. The main criteria is
+  "largely_recommended". Other criteria (for example "reliability", "pedagogy",
+  or "importance") are optional. Users can choose to compare two video on any
+  of the criteria.
+- score:
+  The score is a integer between -10 and +10. Negative values indicate that the
+  user considers the video_a better, and positive values indicate that
+  they prefer the video_b. A score of 0 or close to 0 means that they find the
+  two videos similar.
+- week_date:
+  The date of the Monday of the week during which the comparison was first
+  submitted. Note that it is possible that the comparison is updated at a later
+  time but it is rather rare.
+
 users.csv
 ---------
 
-Type: algorithm input.
-
-This file contains the list of users who appear in the comparisons.csv file.
-
-    trust score:
-
-    The trust score represents how confident the algorithm is that the user is
-    a human with an authentic behavior, using a single account on the platform.
-
-    This score is calculated from the user's email address (trusted domain or
-    not) and from the others users who vouched for them.
-
-    Users with a trust score of 0 (i.e. users without a trusted email and who
-    haven't been vouched for by trusted users) can influence together at most
-    10% of the total collaborative evaluation of a video.
+This file contains the list of users who appear in the comparisons.csv file and
+the trust scores evaluated by Tournesol based on the two mechanisms: 
+1. trusted domain for email addresses
+2. vouching system
+
+List of columns:
+
+- public_username:
+  The public username of the user 
+- trust_score:
+  This field is computed by Tournesol's algorithms. The trust score represents
+  how confident the algorithm is that the user is a human with an authentic
+  behavior and using a single account. Currently this score is calculated based
+  on a list of trusted email domains. Email domains are trusted if there are
+  reasons to believe that this domain only provides a single email address per
+  person. Secondly the trust scores also take into account vouches that any
+  user can choose to give to another account if they trust that the user
+  controlling the account is authentic and controls a single account. Trust
+  scores influence the computation of voting rights (see file
+  individual_criteria_scores.csv).
 
 individual_criteria_scores.csv
 ------------------------------
 
-Type: algorithm output.
```

Comment:
Do we want to remove this notion of input / output in the documentation ?

Reply:
decided on a column basis to specify "This field is computed by Tournesol's algorithms."